### PR TITLE
[*] FO,BO,Project : Improved Tools::normalizeDirectory

### PR DESCRIPTION
**Pull request by andreymishenin**
There are the following problems in the current version of the Tools::normalizeDirectory method that were addressed:
1. Incorrect behavior in case there are several slashes at the end: Tools::normalizeDirectory('/dir//') --> '/dir//'
   Has been fixed to return '/dir/'
2. Not necessary calls to strlen (2 times) and in_array (rather slow function) were removed.

### CODE REVIEW

File: classes/Tools.php [hunk 1]
```diff
@@ -1457,15 +1457,7 @@ public static function truncateString($text, $length = 120, $options = array())
...
-		$last = $directory[strlen($directory) - 1];
-
-		if (in_array($last, array('/', '\\')))
-		{
-			$directory[strlen($directory) - 1] = DIRECTORY_SEPARATOR;
-			return $directory;
-		}
-
-		$directory .= DIRECTORY_SEPARATOR;
+		$directory = $directory = rtrim($directory, '/\\').DIRECTORY_SEPARATOR;;
```
Comment (Shagshag) [inline classes/Tools.php]: Hi,
double ; at the end of the line.

Comment (jeromenadaud) [inline classes/Tools.php]: Hi,

Thank you for your PR, can you also remove double variable assignation ?

Best regards.

Comment (andreymishenin) [inline classes/Tools.php]: Sorry, should I make a new separate commit or a new pull request in case of such a small correction?

**Comment by wellic** [comment]
Fixed

File: classes/Tools.php (hunk 1 above)
Comment (wellic) [inline classes/Tools.php]: Fixed typo in the separate commit.

Comment (wellic) [inline classes/Tools.php]: Removed double variable assignation and double ;
